assets/airports/kbos.json

```diff
@@ -63,12 +65,12 @@
         "_LOGAN22R":  ["N42.30994664062", "W70.92237384752"],
         "_LOGAN27":   ["N42.35182", "W71.03833"],
         "_LOGAN33L":  ["N42.37889", "W71.02394"],
-        "_LOGAN9-1":  ["N42.32617336596", "W70.89029730551"],
-        "_LOGAN9-2":  ["N42.33256591613", "W71.06025996325"],
-        "_LOGAN9-3":  ["N42.40464167947", "W71.08178558128"],
-        "_LOGAN9-4":  ["N42.42750031521", "W70.95315673987"],
-        "_LOGAN9-5":  ["N42.42836031475", "W70.95882644261"],
-        "_LOGAN9-6":  ["N42.32377334760", "W70.90722060934"],
+        "_LOGAN1-1":  ["N42.32617336596", "W70.89029730551"],
+        "_LOGAN1-2":  ["N42.33256591613", "W71.06025996325"],
+        "_LOGAN1-3":  ["N42.40464167947", "W71.08178558128"],
+        "_LOGAN1-4":  ["N42.42750031521", "W70.95315673987"],
+        "_LOGAN1-5":  ["N42.42836031475", "W70.95882644261"],
+        "_LOGAN1-6":  ["N42.32377334760", "W70.90722060934"],
```

Comment:
I believe they were used for the LOGAN1 Departure which I've removed, I'll remove them now.